papa/src/main/java/papa/internal/AppUpdateDetector.kt

```diff
@@ -200,10 +202,21 @@ internal class AppUpdateDetector private constructor(
     }
   }
 
+  @SuppressLint("ApplySharedPref")
+  /**
+   * Records the timestamp of a crash in the application and stores it in shared preferences.
+   * The write operation is performed synchronously to ensure it is completed before the
+   * process terminates in the event of a crash.
+   */
```

Comment:
nit - move the suppresslint annotation to after the comment?

Reply:
Done!